packages/@lwc/wds-core/src/browser/helpers.js

```diff
@@ -70,12 +70,8 @@ export async function clientSideRender(parentEl, componentPath, props = {}, cach
 }
 
 export async function wireMockUtil(mockController) {
-  const setWireValue = async (exportName, data) => {
-    await mockController(`
-        import { createTestWireAdapter } from '${newFileUrl}'; 
-        export const ${exportName} = createTestWireAdapter();
-        ${exportName}.emit(${JSON.stringify(data)});
-    `);
-  };
-  return { setWireValue };
+  await mockController(`
+    import { createTestWireAdapter } from '${newFileUrl}'; 
+    export const myWireAdapter = createTestWireAdapter();
```

Comment:
The name of the export here should be provided as an argument to `wireMockUtil`.

Reply:
sorry missed this